homeassistant/components/mazda/strings.json

```diff
@@ -20,5 +20,96 @@
         "description": "Please enter the email address and password you use to log into the MyMazda mobile app."
       }
     }
+  },
+  "entity": {
+    "binary_sensor": {
+      "driver_door": {
+        "name": "Driver door"
+      },
+      "passenger_door": {
+        "name": "Passenger door"
+      },
+      "rear_left_door": {
+        "name": "Rear left door"
+      },
+      "rear_right_door": {
+        "name": "Rear right door"
+      },
+      "trunk": {
+        "name": "Trunk"
+      },
+      "hood": {
+        "name": "Hood"
+      },
+      "ev_plugged_in": {
+        "name": "Plugged in"
+      }
+    },
+    "button": {
+      "start_engine": {
+        "name": "Start engine"
+      },
+      "stop_engine": {
+        "name": "Stop engine"
+      },
+      "turn_on_hazard_lights": {
+        "name": "Turn on hazard lights"
+      },
+      "turn_off_hazard_lights": {
+        "name": "Turn off hazard lights"
+      },
+      "refresh_vehicle_status": {
+        "name": "Refresh status"
+      }
+    },
+    "climate": {
+      "climate": {
+        "name": "Climate"
+      }
+    },
+    "device_tracker": {
+      "device_tracker": {
+        "name": "Device tracker"
+      }
+    },
+    "lock": {
+      "lock": {
+        "name": "Lock"
+      }
+    },
```

Comment:
These should be references. Also weird that they even need to be implemented on a per-integration basis.